spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/tpcds/datagen/GenSortDataGenerator.java

```diff
@@ -43,12 +45,13 @@
 
     private static final String GENSORT_BIN_DIR_NAME = "bin";
     private static final String GENSORT_DATA_DIR_NAME = "data";
-    private static final String GENERATED_DATA_FILE_NAME = "gensort-data-file";
+    private static final String GENERATED_DATA_FILE_NAME = "gensort_data_file";
 
     private final SparkSession spark;
     private final FileSystem destinationFileSystem;
     private final ParquetTransformer parquetTransformer;
     private final TpcdsPaths paths;
+    private final TpcdsTableRegistration registration;
```

Comment:
maybe we should rename these to `BenchmarkPaths` and `BenchmarkTableRegistration`?

Reply:
Yep going to do a full pass of all the tpcds class names in a follow up (along with the folder structures that are also named with "tpcds")